docs/docs/zh/guide/parameter/built-in.md

```diff
@@ -28,23 +28,33 @@
 - 我们定义这种基准变量为 \$[...] 格式的，\$[yyyyMMddHHmmss] 是可以任意分解组合的，比如：\$[yyyyMMdd], \$[HHmmss], \$[yyyy-MM-dd] 等
 
 - 也可以通过以下两种方式：
-
-  1.使用add_months()函数，该函数用于加减月份，
-  第一个入口参数为[yyyyMMdd]，表示返回时间的格式
-  第二个入口参数为月份偏移量，表示加减多少个月
-  * 后 N 年：$[add_months(yyyyMMdd,12*N)]
-  * 前 N 年：$[add_months(yyyyMMdd,-12*N)]
-  * 后 N 月：$[add_months(yyyyMMdd,N)]
-  * 前 N 月：$[add_months(yyyyMMdd,-N)]
-  *******************************************
-  2.直接加减数字
-  在自定义格式后直接“+/-”数字
-  * 后 N 周：$[yyyyMMdd+7*N]
-  * 前 N 周：$[yyyyMMdd-7*N]
-  * 后 N 天：$[yyyyMMdd+N]
-  * 前 N 天：$[yyyyMMdd-N]
-  * 后 N 小时：$[HHmmss+N/24]
-  * 前 N 小时：$[HHmmss-N/24]
-  * 后 N 分钟：$[HHmmss+N/24/60]
-  * 前 N 分钟：$[HHmmss-N/24/60]
-
+    1.使用add_months()函数，该函数用于加减月份，
+    第一个入口参数为[yyyyMMdd]，表示返回时间的格式
+    第二个入口参数为月份偏移量，表示加减多少个月
```

Comment:
BTW, I think there could be a better design. Instead of using `N` and converting it into different time units, I suggest we use `y` for year, `M` or month, `d` for day. For example, replace `$[add_months(yyyyMMdd,12*N)]` with `$[add(yyyyMMdd, y)]`. But this is out of the scope of this PR. I will create a new issue for further discussion.

Reply:
If my PR is merged, I will consider adding this design to the next PR. However, for business development, people prefer to use independent time expressions rather than one formula to produce multiple time expressions. In other words, I want to calculate the month. I only use addmonth (); I want to calculate the year. I only use addyear () and so on

Reply:
> BTW, I think there could be a better design. Instead of using `N` and converting it into different time units, I suggest we use `y` for year, `M` or month, `d` for day. For example, replace `$[add_months(yyyyMMdd,12*N)]` with `$[add(yyyyMMdd, y)]`. But this is out of the scope of this PR. I will create a new issue for further discussion.

+1

Reply:
> If my PR is merged, I will consider adding this design to the next PR. However, for business development, people prefer to use independent time expressions rather than one formula to produce multiple time expressions. In other words, I want to calculate the month. I only use addmonth (); I want to calculate the year. I only use addyear () and so on

+1